src/components/ebay-breadcrumb/marko-tag.json

```diff
@@ -5,37 +5,31 @@
     "type": "expression"
   },
   "@html-attributes": "expression",
-  "@a11y-heading-text": "string",
-  "@a11y-heading-tag": "string",
-  "@hijax": "boolean",
-  "@aria-labelledby": "never",
-  "@role": "never",
-  "@items <item>[]": {
+  "@index": "string",
```

Comment:
Why has this tag def changed so much?

Reply:
Good catch, I had accidentally copied over the tabs tag instead of the beadcrumb one.